File: katas/6-kyu/highest-scoring-word/solution.js
function high(x){
  let arr = x.split(' ').map(x=>x.split('').reduce((t,x)=>t+x.charCodeAt(0)-96,0))  
  let index =  arr.indexOf(Math.max(... arr) )

    return x.split(' ')[index]
}

File: katas/6-kyu/highest-scoring-word/solution.rs
fn high(input: &str) -> &str {
    let mut max :u32 = 0;
    let mut max_word : &str = "";
    for word in input.split(" "){
        let sum = word.chars().map(|c| c as u32 - 96).sum();
        if sum > max {
            max = sum;
            max_word = word;
        }
    }
    return max_word;
}